### ISSUE TITLE
Adding interactive documentation to the extension panel

### ISSUE THREAD

**Issue by codepo8** [microsoft/vscode-edge-devtools#337]
Instead of a blank document with only the instances running, we should have some docs and call to action buttons to make it easier for developers to get started. See the attached mockup for possible features.

![image](https://user-images.githubusercontent.com/13508/113577140-14ae8180-9621-11eb-9e26-6018cc681000.png)

**Comment by flynnolivia** [comment]
Started taking a look at this on Friday and I came up with a few different views we can show on various conditions:
1. The user doesn't have a folder/workspace open
<img width="252" alt="cta-no-folder" src="https://user-images.githubusercontent.com/80795982/113597566-c1243f80-95f0-11eb-8923-2b2595c874d0.PNG">
2. The user has a folder open, no launch.json detected. Action: create a launch.json with a supported config for our extension
<img width="253" alt="cta-no-json" src="https://user-images.githubusercontent.com/80795982/113597698-f29d0b00-95f0-11eb-99f0-1d691936ca62.PNG">
3. launch.json is detected, but doesn't have a config our extension can use. Action: add a supported config to their launch.json file
<img width="265" alt="cta-unsupported" src="https://user-images.githubusercontent.com/80795982/113597839-1f512280-95f1-11eb-8706-4abb74181498.PNG">
4. launch.json is detected and has a supported config. Action: launch a browser instance using the information from their launch.json. I figure this is where we can automatically start their local server if they have one defined.
<img width="265" alt="cta-supported" src="https://user-images.githubusercontent.com/80795982/113597863-2bd57b00-95f1-11eb-8a3c-a854849b7c83.PNG">

For clarity, the check I've used for seeing if there's a supported config is if they have one where the `type` property is "edge", "msedge", or "vscode-edge-devtools.debug". I also like the idea of linking to the github in the welcome docs, too. Let me know what you think!
